src/index.html

```diff
@@ -13,19 +13,35 @@
     <title>Document</title>
     <link
       rel="stylesheet"
-      href="style.css"
+      href="styles/style.css"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Sen:wght@400..800&display=swap"
+      rel="stylesheet"
     />
   </head>
-  <body>
+  <body class="search">
     <input
+      class="search__big search__big-img"
       type="text"
-      data-qa="keypress"
+      data-qa="keypress big"
       placeholder="Try “Los Angeles“"
+      name="big"
```

Comment:
The attributes for the `input` tag should start on a new line with 2-space indentation related to the tag, as per the checklist for keeping attributes correctly formatted.